README.md

```diff
@@ -45,6 +45,41 @@ __With Docker__
 
 `docker run -it -p 8420:8420 trustwallet/blockatlas`
 
+__Tools__
+- Setup Redis
+
+```shell
+brew install redis // Install Redis using Homebrew
+```
+
+```shell
+ln -sfv /usr/local/opt/redis/*.plist ~/Library/LaunchAgents  // Enable Redis autostart
+```
+
+[Steps Links][https://gist.github.com/nrollr/eb24336b8fb8e7ba5630]
+
+
+- Setup MongoDB
+```shell
+brew install mongodb
+```
+
+```shell
+# Add Mongo DB data directory and set relevant permissions
```

Comment:
what is the purpose of mondo db?

Reply:
@madcake told me it was among the installation steps, so I thought it was the caching layer

Reply:
there is no mongo on blockatlas